src/CollAction/Controllers/ProjectsController.cs

```diff
@@ -108,6 +108,12 @@ public ViewResult StartInfo()
                 return View(createProjectViewModel);
             }
 
+            VideoLink descriptionVideoLink = null;
+            if (createProjectViewModel.DescriptionVideoLink != null)
+            {
+                descriptionVideoLink = new VideoLink { Link = createProjectViewModel.DescriptionVideoLink, Date = DateTime.UtcNow };
+            }
```

Comment:
Small style point (so feel free to ignore, mostly taste). You could do:
VideoLink descriptionVideoLink = 
    createProjectViewModel.DescriptionVideoLink == null ?
        null :
        new VideoLink { Link = createProjectViewModel.DescriptionVideoLink, Date = DateTime.UtcNow };